ROX-18347: CORE_BPF collection method prerequisits (#62570)
openshift/openshift-docs: modules/default-requirements-secured-cluster-services.adoc

Collector monitors runtime activity on each node in your secured clusters. It connects to Sensor to report this information. The collector pod has three containers. The first container is collector, which actually monitors and reports the runtime activity on the node. The other two are compliance and node-inventory.

[discrete]
== Collection requirements

To use the `CORE_BPF` collection method, the base kernel must support BTF, and the BTF file must be available to collector.
In general, the kernel version must be newer than 5.8 (4.18 for RHEL nodes) and the `CONFIG_DEBUG_INFO_BTF` configuration option must be set.

> Collector will look for the BTF file in the following standard locations:

Reviewer: ```suggestion
Collector looks for the BTF file in the standard locations shown in the following list:
```

.BTF file locations
[%collapsible]
====

> ----

Reviewer: ```suggestion
[source,terminal]
----
```

> ----

Reviewer: ```suggestion
[source,terminal]
----
```
